staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -194,10 +194,13 @@ type Config struct {
 	// values below here are targets for removal
 	//===========================================================================
 
-	// PublicAddress is the IP address where members of the cluster (kubelet,
+	// AdvertiseAddress is the IP address where members of the cluster (kubelet,
```

Comment:
Can you update the comments of ExternalAddress above to reflect this rename, i.e. that it defaults to this?